apps/backend/src/common/services/email_service.py

```
@@ -0,0 +1,248 @@
+            # Configure Resend API
+            resend.api_key = auth_config.RESEND_API_KEY
+            
+        # Setup Jinja2 for email templates
+        self.template_env = Environment(
+            loader=FileSystemLoader("src/templates/emails"),
+            autoescape=select_autoescape(["html", "xml"])
+        )
+
+    async def send_email(
+        self,
+        to: List[str],
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)